docs/source/getting_started.rst

```diff
@@ -49,15 +57,19 @@ to your code. See the section below for details.
 Dynamic vs static typing
 ************************
 
+Basedmypy has all strictness options enabled by default. This can often be a
+little overwhelming, and the baseline functionality can help alleviate this.
+For a good compromise of safety and ease of use, we can recommend enabling ``allow_any_expr``.
+
 A function without type annotations is considered to be *dynamically typed* by mypy:
 
 .. code-block:: python
 
    def greeting(name):
        return 'Hello ' + name
 
-By default, mypy will **not** type check dynamically typed functions. This means
-that with a few exceptions, mypy will not report any errors with regular unannotated Python.
+By default, basedmypy **will** type check dynamically typed functions. This means
```

Comment:
i think the bold here is because that behavior is unexpected. if that's no longer the case i dont think it needs to be bolded
```suggestion
By default, basedmypy will type check dynamically typed functions. This means
```